sample/build.gradle

```diff
@@ -0,0 +1,25 @@
+apply plugin: 'com.android.library'
```

Comment:
@friedger non of our other projects have a module called `sample` should this have some common naming?